ci-operator/step-registry/assisted/common/setup/assisted-common-setup-chain.yaml

```diff
@@ -2,7 +2,9 @@ chain:
   as: assisted-common-setup
   steps:
     - ref: ipi-install-rbac
-    - ref: assisted-common-setup
+    - ref: assisted-common-setup-prepare
+    - ref: assisted-common-setup-test
+    - ref: assisted-common-setup-post
```

Comment:
those two are actually part of a test action (``assisted-common-setup-post`` is definitely a test, ``assisted-common-setup-test`` is mixed)
can we structure it in another way?

Reply:
"assisted-common-setup-test" should be "assisted-common-test".
"assisted-common-setup-post" should be "assisted-common-test-post"
just don't want to break it so much... already did a lot of changes.

Reply:
okay